Fix non-unified builds part 2: Missing inlines includes

csaavedra opened the pull request:
#### ec2a4fb830c0784ab58dac2d202fb97543f94f27
<pre>
Fix non-unified builds part 2: Missing inlines includes
<a href="https://bugs.webkit.org/show_bug.cgi?id=295884">https://bugs.webkit.org/show_bug.cgi?id=295884</a>

Unreviewed build fix.

Due to many functions now being in Inlines.h files,
non-unified compilation shows that a lot of those inline
headers are not included in the files where those methods
are used. Include them where appropriate.

As a side fix, document the place of definition of several methods
that were lacking the customary &quot;// Defined in ...&quot; comment.

This is the second of a series of commits trying to fix non-unified
builds, which are severely broken for a while now.

* Source/JavaScriptCore/runtime/Structure.h:
* Source/WebCore/Modules/async-clipboard/ClipboardItemBindingsDataSource.cpp:
* Source/WebCore/Modules/fetch/FetchBody.cpp:
* Source/WebCore/Modules/fetch/FetchBodyOwner.cpp:
* Source/WebCore/Modules/geolocation/Geolocation.cpp:
* Source/WebCore/Modules/mediarecorder/MediaRecorder.cpp:
* Source/WebCore/Modules/mediastream/ImageCapture.cpp:
* Source/WebCore/Modules/mediastream/MediaStream.cpp:
* Source/WebCore/Modules/mediastream/MediaStreamTrack.cpp:
* Source/WebCore/Modules/mediastream/PeerConnectionBackend.cpp:
* Source/WebCore/Modules/mediastream/RTCDataChannel.cpp:
* Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp:
* Source/WebCore/Modules/mediastream/gstreamer/GStreamerPeerConnectionBackend.cpp:
* Source/WebCore/Modules/permissions/PermissionStatus.cpp:
* Source/WebCore/Modules/reporting/ReportingScope.cpp:
* Source/WebCore/Modules/screen-wake-lock/WakeLockSentinel.cpp:
* Source/WebCore/Modules/speech/LocalDOMWindowSpeechSynthesis.cpp:
* Source/WebCore/Modules/speech/SpeechSynthesis.cpp:
* Source/WebCore/Modules/webaudio/AudioWorkletMessagingProxy.cpp:
* Source/WebCore/Modules/webaudio/MediaStreamAudioDestinationNode.cpp:
* Source/WebCore/Modules/webcodecs/WebCodecsBase.cpp:
* Source/WebCore/Modules/webdatabase/DatabaseContext.cpp:
* Source/WebCore/Modules/webxr/WebXRSpace.cpp:
* Source/WebCore/Modules/webxr/WebXRWebGLLayer.cpp:
* Source/WebCore/accessibility/AccessibilityList.cpp:
* Source/WebCore/accessibility/AccessibilityNodeObject.cpp:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
* Source/WebCore/accessibility/AccessibilityScrollView.cpp:
* Source/WebCore/accessibility/AccessibilitySlider.cpp:
* Source/WebCore/accessibility/AccessibilityTable.cpp:
* Source/WebCore/accessibility/atspi/AccessibilityObjectTableAtspi.cpp:
* Source/WebCore/animation/StyleOriginatedAnimation.cpp:
* Source/WebCore/animation/StyleOriginatedAnimationEvent.cpp:
* Source/WebCore/animation/WebAnimation.cpp:
* Source/WebCore/bindings/js/JSCustomElementRegistryCustom.cpp:
* Source/WebCore/bindings/js/JSElementCustom.cpp:
* Source/WebCore/bindings/js/JSRangeCustom.cpp:
* Source/WebCore/bindings/js/ScriptCachedFrameData.cpp:
* Source/WebCore/bindings/js/WindowProxy.cpp:
* Source/WebCore/css/CSSStyleDeclaration.cpp:
* Source/WebCore/css/parser/CSSParser.cpp:
* Source/WebCore/css/query/MediaQueryEvaluator.cpp:
* Source/WebCore/dom/ActiveDOMObject.cpp:
* Source/WebCore/dom/BroadcastChannel.cpp:
* Source/WebCore/dom/ConstantPropertyMap.cpp:
* Source/WebCore/dom/CustomElementDefaultARIA.cpp:
* Source/WebCore/dom/DataTransferItemList.cpp:
* Source/WebCore/dom/Document.h:
* Source/WebCore/dom/ElementTextDirection.cpp:
* Source/WebCore/dom/Event.cpp:
* Source/WebCore/dom/ExtensionStyleSheets.cpp:
* Source/WebCore/dom/Node.h:
* Source/WebCore/dom/PageSwapEvent.cpp:
* Source/WebCore/dom/Position.h:
* Source/WebCore/dom/RejectedPromiseTracker.cpp:
* Source/WebCore/dom/ScriptExecutionContext.cpp:
* Source/WebCore/dom/ScriptExecutionContext.h:
* Source/WebCore/dom/SimulatedClick.cpp:
* Source/WebCore/dom/UserGestureIndicator.cpp:
* Source/WebCore/dom/UserTypingGestureIndicator.cpp:
* Source/WebCore/dom/ViewTransition.cpp:
* Source/WebCore/editing/CompositeEditCommand.cpp:
* Source/WebCore/editing/Editing.cpp:
* Source/WebCore/editing/EditingStyle.cpp:
* Source/WebCore/editing/EditorCommand.cpp:
* Source/WebCore/editing/FrameSelection.cpp:
* Source/WebCore/editing/InsertParagraphSeparatorCommand.cpp:
* Source/WebCore/editing/SimplifyMarkupCommand.cpp:
* Source/WebCore/editing/TextCheckingHelper.cpp:
* Source/WebCore/editing/TextManipulationController.cpp:
* Source/WebCore/editing/VisiblePosition.cpp:
* Source/WebCore/editing/WebContentReader.cpp:
* Source/WebCore/editing/glib/EditorGLib.cpp:
* Source/WebCore/editing/glib/WebContentReaderGLib.cpp:
* Source/WebCore/fileapi/Blob.cpp:
* Source/WebCore/history/CachedPage.cpp:
* Source/WebCore/html/BaseCheckableInputType.cpp:
* Source/WebCore/html/CanvasBase.cpp:
* Source/WebCore/html/ColorInputType.cpp:
* Source/WebCore/html/CustomPaintImage.cpp:
* Source/WebCore/html/DOMFormData.cpp:
* Source/WebCore/html/DOMTokenList.cpp:
* Source/WebCore/html/HTMLAnchorElement.cpp:
* Source/WebCore/html/HTMLCanvasElement.cpp:
* Source/WebCore/html/HTMLDialogElement.cpp:
* Source/WebCore/html/HTMLFormControlElement.cpp:
* Source/WebCore/html/HTMLFrameElement.cpp:
* Source/WebCore/html/HTMLFrameElementBase.cpp:
* Source/WebCore/html/HTMLFrameOwnerElement.cpp:
* Source/WebCore/html/HTMLFrameSetElement.cpp:
* Source/WebCore/html/HTMLMarqueeElement.cpp:
* Source/WebCore/html/HTMLObjectElement.cpp:
* Source/WebCore/html/HTMLOptionElement.cpp:
* Source/WebCore/html/HTMLPlugInImageElement.cpp:
* Source/WebCore/html/HTMLProgressElement.cpp:
* Source/WebCore/html/HTMLStyleElement.cpp:
* Source/WebCore/html/HTMLTableColElement.cpp:
* Source/WebCore/html/HTMLTextFormControlElement.cpp:
* Source/WebCore/html/ImageBitmap.cpp:
* Source/WebCore/html/ImageDocument.cpp:
* Source/WebCore/html/ImageInputType.cpp:
* Source/WebCore/html/MediaController.cpp:
* Source/WebCore/html/NumberInputType.cpp:
* Source/WebCore/html/PluginDocument.cpp:
* Source/WebCore/html/SearchInputType.cpp:
* Source/WebCore/html/ValidationMessage.cpp:
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
* Source/WebCore/html/canvas/CanvasStyle.cpp:
* Source/WebCore/html/canvas/EXTDisjointTimerQuery.cpp:
* Source/WebCore/html/canvas/EXTDisjointTimerQueryWebGL2.cpp:
* Source/WebCore/html/canvas/PlaceholderRenderingContext.cpp:
* Source/WebCore/html/parser/HTMLScriptRunner.cpp:
* Source/WebCore/html/shadow/DateTimeEditElement.cpp:
* Source/WebCore/html/shadow/MediaControlTextTrackContainerElement.cpp:
* Source/WebCore/html/shadow/ProgressShadowElement.cpp:
* Source/WebCore/html/shadow/SliderThumbElement.cpp:
* Source/WebCore/html/shadow/SpinButtonElement.cpp:
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
* Source/WebCore/html/track/InbandGenericTextTrack.cpp:
* Source/WebCore/html/track/TextTrackCueGeneric.cpp:
* Source/WebCore/inspector/DOMEditor.cpp:
* Source/WebCore/inspector/InspectorFrontendHost.cpp:
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
* Source/WebCore/inspector/InspectorStyleSheet.cpp:
* Source/WebCore/inspector/PageDebugger.cpp:
* Source/WebCore/inspector/agents/InspectorAnimationAgent.cpp:
* Source/WebCore/inspector/agents/InspectorDOMStorageAgent.cpp:
* Source/WebCore/inspector/agents/InspectorIndexedDBAgent.cpp:
* Source/WebCore/inspector/agents/InspectorLayerTreeAgent.cpp:
* Source/WebCore/inspector/agents/InspectorPageAgent.cpp:
* Source/WebCore/inspector/agents/page/PageCanvasAgent.cpp:
* Source/WebCore/layout/integration/LayoutIntegrationBoxTreeUpdater.cpp:
* Source/WebCore/layout/integration/LayoutIntegrationCoverage.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationLineLayout.cpp:
* Source/WebCore/loader/FrameLoadRequest.cpp:
* Source/WebCore/loader/FrameMemoryMonitor.cpp:
* Source/WebCore/loader/ResourceMonitor.cpp:
* Source/WebCore/loader/SubresourceIntegrity.cpp:
* Source/WebCore/loader/cache/CachedResourceRequest.cpp:
* Source/WebCore/mathml/MathMLMathElement.cpp:
* Source/WebCore/mathml/MathMLOperatorElement.cpp:
* Source/WebCore/mathml/MathMLSelectElement.cpp:
* Source/WebCore/mathml/MathMLTokenElement.cpp:
* Source/WebCore/page/DOMWindow.cpp:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/FocusController.cpp:
* Source/WebCore/page/Frame.cpp:
* Source/WebCore/page/FrameSnapshotting.cpp:
* Source/WebCore/page/FrameTree.cpp:
* Source/WebCore/page/FrameView.cpp:
* Source/WebCore/page/ImageOverlayController.cpp:
* Source/WebCore/page/IntersectionObserver.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/NavigateEvent.cpp:
* Source/WebCore/page/PageColorSampler.cpp:
* Source/WebCore/page/PageGroupLoadDeferrer.cpp:
* Source/WebCore/page/Performance.cpp:
* Source/WebCore/page/PerformanceTiming.cpp:
* Source/WebCore/page/PrintContext.cpp:
* Source/WebCore/page/RemoteFrame.cpp:
* Source/WebCore/page/SecurityOriginData.cpp:
* Source/WebCore/page/TextIndicator.cpp:
* Source/WebCore/page/UndoManager.cpp:
* Source/WebCore/page/scrolling/ScrollSnapOffsetsInfo.cpp:
* Source/WebCore/plugins/PluginData.cpp:
* Source/WebCore/rendering/CSSFilter.cpp:
* Source/WebCore/rendering/ImageQualityController.cpp:
* Source/WebCore/rendering/LegacyInlineBox.cpp:
* Source/WebCore/rendering/MarkedText.cpp:
* Source/WebCore/rendering/MotionPath.cpp:
* Source/WebCore/rendering/RenderBlock.cpp:
* Source/WebCore/rendering/RenderBox.cpp:
* Source/WebCore/rendering/RenderElement.cpp:
* Source/WebCore/rendering/RenderElement.h:
* Source/WebCore/rendering/RenderFileUploadControl.cpp:
* Source/WebCore/rendering/RenderLayerModelObject.cpp:
* Source/WebCore/rendering/RenderLayerScrollableArea.cpp:
* Source/WebCore/rendering/RenderListItem.cpp:
* Source/WebCore/rendering/RenderMarquee.cpp:
* Source/WebCore/rendering/RenderObject.cpp:
* Source/WebCore/rendering/RenderProgress.cpp:
* Source/WebCore/rendering/RenderQuote.cpp:
* Source/WebCore/rendering/RenderScrollbar.cpp:
* Source/WebCore/rendering/RenderSearchField.cpp:
* Source/WebCore/rendering/RenderTextControlSingleLine.cpp:
* Source/WebCore/rendering/RenderTextFragment.cpp:
* Source/WebCore/rendering/RenderTheme.cpp:
* Source/WebCore/rendering/RenderTreeAsText.cpp:
* Source/WebCore/rendering/mathml/MathMLStyle.cpp:
* Source/WebCore/rendering/mathml/RenderMathMLFraction.cpp:
* Source/WebCore/rendering/mathml/RenderMathMLSpace.cpp:
* Source/WebCore/rendering/style/StyleCanvasImage.cpp:
* Source/WebCore/rendering/style/StyleGeneratedImage.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.cpp:
* Source/WebCore/rendering/svg/SVGRenderingContext.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/legacy/SVGResources.cpp:
* Source/WebCore/rendering/svg/legacy/SVGResourcesCache.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilderRuby.cpp:
* Source/WebCore/rendering/updating/RenderTreePosition.cpp:
* Source/WebCore/storage/StorageNamespaceProvider.cpp:
* Source/WebCore/style/ContainerQueryEvaluator.cpp:
* Source/WebCore/style/StyleFontSizeFunctions.cpp:
* Source/WebCore/style/StyleUpdate.cpp:
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/svg/SVGDocumentExtensions.cpp:
* Source/WebCore/svg/SVGFEDropShadowElement.cpp:
* Source/WebCore/svg/SVGFEFloodElement.cpp:
* Source/WebCore/svg/SVGFELightElement.cpp:
* Source/WebCore/svg/SVGFilterElement.cpp:
* Source/WebCore/svg/SVGFitToViewBox.cpp:
* Source/WebCore/svg/SVGGeometryElement.cpp:
* Source/WebCore/svg/SVGGradientElement.cpp:
* Source/WebCore/svg/SVGGraphicsElement.cpp:
* Source/WebCore/svg/SVGLengthContext.cpp:
* Source/WebCore/svg/SVGLengthValue.cpp:
* Source/WebCore/svg/SVGLineElement.cpp:
* Source/WebCore/svg/SVGLinearGradientElement.cpp:
* Source/WebCore/svg/SVGMarkerElement.cpp:
* Source/WebCore/svg/SVGMaskElement.cpp:
* Source/WebCore/svg/SVGPolyElement.cpp:
* Source/WebCore/svg/SVGTextPathElement.cpp:
* Source/WebCore/svg/SVGTextPositioningElement.cpp:
* Source/WebCore/svg/properties/SVGAnimatedString.cpp:
* Source/WebCore/workers/service/ServiceWorkerContainer.cpp:
* Source/WebCore/workers/service/background-fetch/BackgroundFetchRegistration.cpp:
* Source/WebCore/worklets/Worklet.cpp:
* Source/WebCore/xml/XPathExpression.cpp:
* Source/WebCore/xml/XSLTProcessor.cpp:
* Source/WebKit/WebProcess/Network/WebResourceLoader.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebContextMenuClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebRemoteFrameClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/wpe/WebEditorClientWPE.cpp:
* Source/WebKit/WebProcess/WebPage/WebContextMenu.cpp:

Canonical link: <a href="https://commits.webkit.org/297334@main">https://commits.webkit.org/297334@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/06a18a9024f979f074a1c919b481f5977068ae44

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/111430 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/31096 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/21561 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/117462 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/61698 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/113392 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/31777 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/39678 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/84691 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/114377 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/25376 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/100312 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/65138 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/24723 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/18448 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/61282 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/103923 "Built successfully and passed tests") | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/94763 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/18514 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/120687 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/109984 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/38479 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/28599 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/93617 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/38855 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/96581 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/93442 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23799 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/38546 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/16321 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/34513 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/38368 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/43845 "Built successfully") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/134259 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/38033 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/36186 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/41366 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/39735 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/06a18a9024f979f074a1c919b481f5977068ae44. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/47974

csaavedra committed:
Fix non-unified builds part 2: Missing inlines includes
https://bugs.webkit.org/show_bug.cgi?id=295884

Unreviewed build fix.

Due to many functions now being in Inlines.h files,
non-unified compilation shows that a lot of those inline
headers are not included in the files where those methods
are used. Include them where appropriate.

As a side fix, document the place of definition of several methods
that were lacking the customary "// Defined in ..." comment.

This is the second of a series of commits trying to fix non-unified
builds, which are severely broken for a while now.

* Source/JavaScriptCore/runtime/Structure.h:
* Source/WebCore/Modules/async-clipboard/ClipboardItemBindingsDataSource.cpp:
* Source/WebCore/Modules/fetch/FetchBody.cpp:
* Source/WebCore/Modules/fetch/FetchBodyOwner.cpp:
* Source/WebCore/Modules/geolocation/Geolocation.cpp:
* Source/WebCore/Modules/mediarecorder/MediaRecorder.cpp:
* Source/WebCore/Modules/mediastream/ImageCapture.cpp:
* Source/WebCore/Modules/mediastream/MediaStream.cpp:
* Source/WebCore/Modules/mediastream/MediaStreamTrack.cpp:
* Source/WebCore/Modules/mediastream/PeerConnectionBackend.cpp:
* Source/WebCore/Modules/mediastream/RTCDataChannel.cpp:
* Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp:
* Source/WebCore/Modules/mediastream/gstreamer/GStreamerPeerConnectionBackend.cpp:
* Source/WebCore/Modules/permissions/PermissionStatus.cpp:
* Source/WebCore/Modules/reporting/ReportingScope.cpp:
* Source/WebCore/Modules/screen-wake-lock/WakeLockSentinel.cpp:
* Source/WebCore/Modules/speech/LocalDOMWindowSpeechSynthesis.cpp:
* Source/WebCore/Modules/speech/SpeechSynthesis.cpp:
* Source/WebCore/Modules/webaudio/AudioWorkletMessagingProxy.cpp:
* Source/WebCore/Modules/webaudio/MediaStreamAudioDestinationNode.cpp:
* Source/WebCore/Modules/webcodecs/WebCodecsBase.cpp:
* Source/WebCore/Modules/webdatabase/DatabaseContext.cpp:
* Source/WebCore/Modules/webxr/WebXRSpace.cpp:
* Source/WebCore/Modules/webxr/WebXRWebGLLayer.cpp:
* Source/WebCore/accessibility/AccessibilityList.cpp:
* Source/WebCore/accessibility/AccessibilityNodeObject.cpp:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
* Source/WebCore/accessibility/AccessibilityScrollView.cpp:
* Source/WebCore/accessibility/AccessibilitySlider.cpp:
* Source/WebCore/accessibility/AccessibilityTable.cpp:
* Source/WebCore/accessibility/atspi/AccessibilityObjectTableAtspi.cpp:
* Source/WebCore/animation/StyleOriginatedAnimation.cpp:
* Source/WebCore/animation/StyleOriginatedAnimationEvent.cpp:
* Source/WebCore/animation/WebAnimation.cpp:
* Source/WebCore/bindings/js/JSCustomElementRegistryCustom.cpp:
* Source/WebCore/bindings/js/JSElementCustom.cpp:
* Source/WebCore/bindings/js/JSRangeCustom.cpp:
* Source/WebCore/bindings/js/ScriptCachedFrameData.cpp:
* Source/WebCore/bindings/js/WindowProxy.cpp:
* Source/WebCore/css/CSSStyleDeclaration.cpp:
* Source/WebCore/css/parser/CSSParser.cpp:
* Source/WebCore/css/query/MediaQueryEvaluator.cpp:
* Source/WebCore/dom/ActiveDOMObject.cpp:
* Source/WebCore/dom/BroadcastChannel.cpp:
* Source/WebCore/dom/ConstantPropertyMap.cpp:
* Source/WebCore/dom/CustomElementDefaultARIA.cpp:
* Source/WebCore/dom/DataTransferItemList.cpp:
* Source/WebCore/dom/Document.h:
* Source/WebCore/dom/ElementTextDirection.cpp:
* Source/WebCore/dom/Event.cpp:
* Source/WebCore/dom/ExtensionStyleSheets.cpp:
* Source/WebCore/dom/Node.h:
* Source/WebCore/dom/PageSwapEvent.cpp:
* Source/WebCore/dom/Position.h:
* Source/WebCore/dom/RejectedPromiseTracker.cpp:
* Source/WebCore/dom/ScriptExecutionContext.cpp:
* Source/WebCore/dom/ScriptExecutionContext.h:
* Source/WebCore/dom/SimulatedClick.cpp:
* Source/WebCore/dom/UserGestureIndicator.cpp:
* Source/WebCore/dom/UserTypingGestureIndicator.cpp:
* Source/WebCore/dom/ViewTransition.cpp:
* Source/WebCore/editing/CompositeEditCommand.cpp:
* Source/WebCore/editing/Editing.cpp:
* Source/WebCore/editing/EditingStyle.cpp:
* Source/WebCore/editing/EditorCommand.cpp:
* Source/WebCore/editing/FrameSelection.cpp:
* Source/WebCore/editing/InsertParagraphSeparatorCommand.cpp:
* Source/WebCore/editing/SimplifyMarkupCommand.cpp:
* Source/WebCore/editing/TextCheckingHelper.cpp:
* Source/WebCore/editing/TextManipulationController.cpp:
* Source/WebCore/editing/VisiblePosition.cpp:
* Source/WebCore/editing/WebContentReader.cpp:
* Source/WebCore/editing/glib/EditorGLib.cpp:
* Source/WebCore/editing/glib/WebContentReaderGLib.cpp:
* Source/WebCore/fileapi/Blob.cpp:
* Source/WebCore/history/CachedPage.cpp:
* Source/WebCore/html/BaseCheckableInputType.cpp:
* Source/WebCore/html/CanvasBase.cpp:
* Source/WebCore/html/ColorInputType.cpp:
* Source/WebCore/html/CustomPaintImage.cpp:
* Source/WebCore/html/DOMFormData.cpp:
* Source/WebCore/html/DOMTokenList.cpp:
* Source/WebCore/html/HTMLAnchorElement.cpp:
* Source/WebCore/html/HTMLCanvasElement.cpp:
* Source/WebCore/html/HTMLDialogElement.cpp:
* Source/WebCore/html/HTMLFormControlElement.cpp:
* Source/WebCore/html/HTMLFrameElement.cpp:
* Source/WebCore/html/HTMLFrameElementBase.cpp:
* Source/WebCore/html/HTMLFrameOwnerElement.cpp:
* Source/WebCore/html/HTMLFrameSetElement.cpp:
* Source/WebCore/html/HTMLMarqueeElement.cpp:
* Source/WebCore/html/HTMLObjectElement.cpp:
* Source/WebCore/html/HTMLOptionElement.cpp:
* Source/WebCore/html/HTMLPlugInImageElement.cpp:
* Source/WebCore/html/HTMLProgressElement.cpp:
* Source/WebCore/html/HTMLStyleElement.cpp:
* Source/WebCore/html/HTMLTableColElement.cpp:
* Source/WebCore/html/HTMLTextFormControlElement.cpp:
* Source/WebCore/html/ImageBitmap.cpp:
* Source/WebCore/html/ImageDocument.cpp:
* Source/WebCore/html/ImageInputType.cpp:
* Source/WebCore/html/MediaController.cpp:
* Source/WebCore/html/NumberInputType.cpp:
* Source/WebCore/html/PluginDocument.cpp:
* Source/WebCore/html/SearchInputType.cpp:
* Source/WebCore/html/ValidationMessage.cpp:
* Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp:
* Source/WebCore/html/canvas/CanvasStyle.cpp:
* Source/WebCore/html/canvas/EXTDisjointTimerQuery.cpp:
* Source/WebCore/html/canvas/EXTDisjointTimerQueryWebGL2.cpp:
* Source/WebCore/html/canvas/PlaceholderRenderingContext.cpp:
* Source/WebCore/html/parser/HTMLScriptRunner.cpp:
* Source/WebCore/html/shadow/DateTimeEditElement.cpp:
* Source/WebCore/html/shadow/MediaControlTextTrackContainerElement.cpp:
* Source/WebCore/html/shadow/ProgressShadowElement.cpp:
* Source/WebCore/html/shadow/SliderThumbElement.cpp:
* Source/WebCore/html/shadow/SpinButtonElement.cpp:
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
* Source/WebCore/html/track/InbandGenericTextTrack.cpp:
* Source/WebCore/html/track/TextTrackCueGeneric.cpp:
* Source/WebCore/inspector/DOMEditor.cpp:
* Source/WebCore/inspector/InspectorFrontendHost.cpp:
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
* Source/WebCore/inspector/InspectorStyleSheet.cpp:
* Source/WebCore/inspector/PageDebugger.cpp:
* Source/WebCore/inspector/agents/InspectorAnimationAgent.cpp:
* Source/WebCore/inspector/agents/InspectorDOMStorageAgent.cpp:
* Source/WebCore/inspector/agents/InspectorIndexedDBAgent.cpp:
* Source/WebCore/inspector/agents/InspectorLayerTreeAgent.cpp:
* Source/WebCore/inspector/agents/InspectorPageAgent.cpp:
* Source/WebCore/inspector/agents/page/PageCanvasAgent.cpp:
* Source/WebCore/layout/integration/LayoutIntegrationBoxTreeUpdater.cpp:
* Source/WebCore/layout/integration/LayoutIntegrationCoverage.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationLineLayout.cpp:
* Source/WebCore/loader/FrameLoadRequest.cpp:
* Source/WebCore/loader/FrameMemoryMonitor.cpp:
* Source/WebCore/loader/ResourceMonitor.cpp:
* Source/WebCore/loader/SubresourceIntegrity.cpp:
* Source/WebCore/loader/cache/CachedResourceRequest.cpp:
* Source/WebCore/mathml/MathMLMathElement.cpp:
* Source/WebCore/mathml/MathMLOperatorElement.cpp:
* Source/WebCore/mathml/MathMLSelectElement.cpp:
* Source/WebCore/mathml/MathMLTokenElement.cpp:
* Source/WebCore/page/DOMWindow.cpp:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/FocusController.cpp:
* Source/WebCore/page/Frame.cpp:
* Source/WebCore/page/FrameSnapshotting.cpp:
* Source/WebCore/page/FrameTree.cpp:
* Source/WebCore/page/FrameView.cpp:
* Source/WebCore/page/ImageOverlayController.cpp:
* Source/WebCore/page/IntersectionObserver.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/NavigateEvent.cpp:
* Source/WebCore/page/PageColorSampler.cpp:
* Source/WebCore/page/PageGroupLoadDeferrer.cpp:
* Source/WebCore/page/Performance.cpp:
* Source/WebCore/page/PerformanceTiming.cpp:
* Source/WebCore/page/PrintContext.cpp:
* Source/WebCore/page/RemoteFrame.cpp:
* Source/WebCore/page/SecurityOriginData.cpp:
* Source/WebCore/page/TextIndicator.cpp:
* Source/WebCore/page/UndoManager.cpp:
* Source/WebCore/page/scrolling/ScrollSnapOffsetsInfo.cpp:
* Source/WebCore/plugins/PluginData.cpp:
* Source/WebCore/rendering/CSSFilter.cpp:
* Source/WebCore/rendering/ImageQualityController.cpp:
* Source/WebCore/rendering/LegacyInlineBox.cpp:
* Source/WebCore/rendering/MarkedText.cpp:
* Source/WebCore/rendering/MotionPath.cpp:
* Source/WebCore/rendering/RenderBlock.cpp:
* Source/WebCore/rendering/RenderBox.cpp:
* Source/WebCore/rendering/RenderElement.cpp:
* Source/WebCore/rendering/RenderElement.h:
* Source/WebCore/rendering/RenderFileUploadControl.cpp:
* Source/WebCore/rendering/RenderLayerModelObject.cpp:
* Source/WebCore/rendering/RenderLayerScrollableArea.cpp:
* Source/WebCore/rendering/RenderListItem.cpp:
* Source/WebCore/rendering/RenderMarquee.cpp:
* Source/WebCore/rendering/RenderObject.cpp:
* Source/WebCore/rendering/RenderProgress.cpp:
* Source/WebCore/rendering/RenderQuote.cpp:
* Source/WebCore/rendering/RenderScrollbar.cpp:
* Source/WebCore/rendering/RenderSearchField.cpp:
* Source/WebCore/rendering/RenderTextControlSingleLine.cpp:
* Source/WebCore/rendering/RenderTextFragment.cpp:
* Source/WebCore/rendering/RenderTheme.cpp:
* Source/WebCore/rendering/RenderTreeAsText.cpp:
* Source/WebCore/rendering/mathml/MathMLStyle.cpp:
* Source/WebCore/rendering/mathml/RenderMathMLFraction.cpp:
* Source/WebCore/rendering/mathml/RenderMathMLSpace.cpp:
* Source/WebCore/rendering/style/StyleCanvasImage.cpp:
* Source/WebCore/rendering/style/StyleGeneratedImage.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.cpp:
* Source/WebCore/rendering/svg/SVGRenderingContext.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceContainer.cpp:
* Source/WebCore/rendering/svg/legacy/SVGResources.cpp:
* Source/WebCore/rendering/svg/legacy/SVGResourcesCache.cpp:
* Source/WebCore/rendering/updating/RenderTreeBuilderRuby.cpp:
* Source/WebCore/rendering/updating/RenderTreePosition.cpp:
* Source/WebCore/storage/StorageNamespaceProvider.cpp:
* Source/WebCore/style/ContainerQueryEvaluator.cpp:
* Source/WebCore/style/StyleFontSizeFunctions.cpp:
* Source/WebCore/style/StyleUpdate.cpp:
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/svg/SVGDocumentExtensions.cpp:
* Source/WebCore/svg/SVGFEDropShadowElement.cpp:
* Source/WebCore/svg/SVGFEFloodElement.cpp:
* Source/WebCore/svg/SVGFELightElement.cpp:
* Source/WebCore/svg/SVGFilterElement.cpp:
* Source/WebCore/svg/SVGFitToViewBox.cpp:
* Source/WebCore/svg/SVGGeometryElement.cpp:
* Source/WebCore/svg/SVGGradientElement.cpp:
* Source/WebCore/svg/SVGGraphicsElement.cpp:
* Source/WebCore/svg/SVGLengthContext.cpp:
* Source/WebCore/svg/SVGLengthValue.cpp:
* Source/WebCore/svg/SVGLineElement.cpp:
* Source/WebCore/svg/SVGLinearGradientElement.cpp:
* Source/WebCore/svg/SVGMarkerElement.cpp:
* Source/WebCore/svg/SVGMaskElement.cpp:
* Source/WebCore/svg/SVGPolyElement.cpp:
* Source/WebCore/svg/SVGTextPathElement.cpp:
* Source/WebCore/svg/SVGTextPositioningElement.cpp:
* Source/WebCore/svg/properties/SVGAnimatedString.cpp:
* Source/WebCore/workers/service/ServiceWorkerContainer.cpp:
* Source/WebCore/workers/service/background-fetch/BackgroundFetchRegistration.cpp:
* Source/WebCore/worklets/Worklet.cpp:
* Source/WebCore/xml/XPathExpression.cpp:
* Source/WebCore/xml/XSLTProcessor.cpp:
* Source/WebKit/WebProcess/Network/WebResourceLoader.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebContextMenuClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebRemoteFrameClient.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/wpe/WebEditorClientWPE.cpp:
* Source/WebKit/WebProcess/WebPage/WebContextMenu.cpp:

Canonical link: https://commits.webkit.org/297334@main
